docs-site/blog/2020-06-10-new-site.md

```diff
@@ -0,0 +1,24 @@
+---
+id: new-site
+title: A new docs site, and recent updates
+author: Ashwin Ramaswami
+author_title: CodaLab developer
+author_url: https://github.com/epicfaace
+author_image_url: https://avatars3.githubusercontent.com/u/1689183
+tags: [introduction, release, blog]
+---
+
+We're pleased to announce a new documentation site, built with [Docusaurus](https://v2.docusaurus.io/). We'll also be sending out an occasional email digest with the updates that are posted to the blog here. If you're interested in being added to the list, please email <a href="mailto:codalab.worksheets@gmail.com">codalab.worksheets@gmail.com</a>.
+
+Here are some of the major new features that we have released within the past few months:
+
+## Frontend improvements
+
+- We've introduced a new frontend with drag and drop options. We also have a new UI for bundle detail layout ([#1543](https://github.com/codalab/codalab-worksheets/pull/1543))
```

Comment:
I think the drag and drop should be removed. Do we need to mention the detail layout? It seems to be a long time ago